### ISSUE TITLE
RPC fail error occurs frequently

### ISSUE THREAD

**Issue by mrliliang** [Hacker0912/RecStep#6]
When a datalog program has two many iterations, RecStep terminates with error RPC fail. I am not sure whether it is caused by RecStep or QuickStep database.

<img width="941" alt="image" src="https://user-images.githubusercontent.com/12110905/224263851-2f87f017-9fec-445a-a9d1-93d93b686441.png">

**Comment by Hacker0912** [comment]
Without digging in further, I think this is tied to the emerging bug we observed in `QuickStep` a while ago and you might skip that note in the readme (I could be wrong if it was caused by other issues) - one very quick thing to try is to set `dynamic_set_diff` to be false in `Config.json` - the performance might not be optimal, depending on the actual input data but it should not suffer too much in most cases. Unfortunately the QuickStep team is gone and nobody is actively maintaining QuickStep anymore - many of its third-party dependencies could be broken now and then. I tried to "lock" a relatively stable version but it turned out to show some issues still.
